packages/sources/snowflake/src/endpoint/covid-cases.ts

```diff
@@ -0,0 +1,64 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
+import { buildSnowflakeJWT } from '../util'
+
+export const supportedEndpoints = ['covid-cases']
+
+export const inputParameters: InputParameters = {
+  state: ['state'],
+  county: ['county'],
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, _, config) => {
+  const validator = new Validator(request, inputParameters)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const state = validator.validated.data.state
+  const county = validator.validated.data.county
+
+  const jwtToken = buildSnowflakeJWT({
+    privateKey: config.api.privateKey,
+    qualifiedUsername: config.api.qualifiedUsername,
+  })
+
+  const response = await Requester.request({
+    baseURL: config.api.baseURL,
+    url: '/statements',
+    method: 'POST',
+    data: {
+      database: config.api.database,
+      schema: config.api.schema,
+      statement: `
+        select confirmed
+        from JHU_DASHBOARD_COVID_19_GLOBAL
+        where country_region = 'United States'
+        and province_state = :1
+        and county = :2
+        limit 1
```

Comment:
Since we're doing a `limit 1` do we also need to have an `order by` to ensure we get the correct first result?

Reply:
The combination of `province_state` and `county` is unique, so the limit wasn't technically necessary, removed in new rev.